shipment.go

```diff
@@ -290,6 +290,21 @@ func (c *Client) GetShipmentRatesWithContext(ctx context.Context, shipmentID str
 	return
 }
 
+// GetShipmentSmartrates fetches the available smartrates for a shipment.
+func (c *Client) GetShipmentSmartrates(shipmentID string) (out []*Rate, err error) {
```

Comment:
My spidy senses tell me that we should capitalize `SmartRates` to be fully CamelCased but I could be convinced otherwise.

Reply:
I haven't in other libraries, in my mind, `Smartrate` is a single object/word/thought

Reply:
i don't have any strong feelings eitherway so i'll leave this up to your discretion